src/dirs.rs

```diff
@@ -1,21 +1,48 @@
+#[cfg(not(target_os="android"))]
```

Comment:
It's possible to cfg the whole module, do that instead so you don't have to do return Err(());

Reply:
Yeah but then all functions that use dirs would break, and I believe some paths should be returned here on Android, I just don't know which yet.